packages/design-systems/src/components/composite/Datagrid/SearchFilter/FilterRow.tsx

```diff
@@ -332,10 +332,10 @@ export const FilterRow = <TData extends Record<string, unknown>>(
             data-testid="select-column-options"
           >
             <Select.ItemGroup className={classes.itemGroup} id="column">
-              {columns.map((item) => (
+              {columns.map((item, i) => (
                 <Select.Item
                   className={classes.item}
-                  key={item.value}
+                  key={item.value + i}
```

Comment:
This is the cause of the ID duplicate error.

Reply:
ここが原因なら、ここ以外の `id` の箇所は修正不要かと思ったんですが、違うんですかね?

Reply:
直接的にはここで発生していましたが、他の箇所でも同様の理由で発生する可能性がある箇所だったので合わせて修正をしておきました。

Reply:
なるほど。

ちょっと考えたんですが、全ての `key` に対する実装でindexのみを使うようにしてもらいたいです。`id` や `value` はユーザー定義で変更できるので、重複するリスクがあるということはわかりました。なので、そもそも使う必要がないかなと。

```suggestion
                  key={i}
```

Reply:
確かにそうですね。他の箇所も含めて修正します。